docker-storage-setup.conf

```diff
@@ -84,3 +83,28 @@ DEVICE_WAIT_TIMEOUT=60
 # by setting WIPE_SIGNATURES=true. Be careful before using this option
 # as this means if there was any leftover data on disk, it will be lost.
 WIPE_SIGNATURES=false
+
+# By default no new volume and filesystem (fs) will be setup for docker
+# root dir. Docker will create /var/lib/docker/ and use underlying fs for
+# storing image and containers.
+#
+# If one wants to carve out a separate logical volume for storing docker
+# metadata and images/containers/volumes, then set DOCKER_ROOT_VOLUME=yes
+#
+# This will try to create a logical volume, create a XFS filesystem on
+# that volume and mount on $DOCKER_ROOT_DIR.
+#
+# NOTE: If both STORAGE_DRIVER=devicemapper and DOCKER_ROOT_VOLUME=yes is
+# passed, docker-storage-setup would set up the thin pool for devicemapper
+# first, followed by docker root volume.
+DOCKER_ROOT_VOLUME=no
+
+# The desired size for the docker root volume. It defaults using 40% of all
```

Comment:
Shouldn't this start out with a 
DOCKER_ROOT_VOLUME_SIZE

Is this the Percentage of free space available 
The default size for DOCKER_ROOT_VOLUME_SIZE